test/unit/helper/StateWidgetTest.php

```diff
@@ -46,10 +46,14 @@ class StateWidgetTest extends TestCase
     private const TRANSITION_ID_1 = 'id_1';
     private const TRANSITION_ID_2 = 'id_2';
 
-    /** @var JsonWorkflow|MockObject  */
+    /**
+     * @var JsonWorkflow|MockObject
+     */
```

Comment:
```suggestion
    /** @var JsonWorkflow|MockObject */
```

We actually prefer to use like this for this cases https://github.com/oat-sa/extension-tao-resource-workflow/pull/45/files#r1058759119